tests/test_bot.rs

```diff
@@ -23,7 +23,6 @@ mod tests {
         (token, chat_id)
     }
 
-    #[test]
```

Comment:
sorry for that, I had to take those away for testing because I was getting errors from telegram saying I am making too many requests.